torch/nn/utils/parametrize.py

```diff
@@ -66,17 +66,21 @@ class ParametrizationList(ModuleList):
     Args:
         modules (iterable): an iterable of modules representing the parametrizations
         original (Parameter or Tensor): parameter or buffer that is parametrized
+        allow_resize (bool): a boolean flag that denotes whether the parametrized tensor can be resized,
+            Warning: the current Module might not work anymore, enable this flag at the user's own risk.
```

Comment:
The person reading this is the user :D 
So, "enable this flag at your own risk"